@mhajas I am running the tests verbosely using `mvn clean install -f testsuite/integration-arquillian/pom.xml -Dtest=JavascriptAdapterTest -X` and I can't make heads or tails of what is failing (see below).


<details><summary>Logs</summary>
<p>

```
[INFO] Running org.keycloak.testsuite.javascript.JavascriptAdapterTest
Sep 10, 2019 4:42:47 PM org.keycloak.testsuite.arquillian.containers.MultipleContainersExtension register
INFO: Multiple containers extension registering.
16:42:47,547 INFO  [org.keycloak.testsuite.arquillian.containers.RegistryCreator] arquillian.xml: /home/jon/Projects/personal/keycloak/testsuite/integration-arquillian/tests/base/target/dependency/arquillian.xml
16:42:47,683 INFO  [org.keycloak.testsuite.arquillian.containers.RegistryCreator] Registering container: auth-server-undertow
16:42:47,686 WARN  [org.keycloak.testsuite.arquillian.containers.RegistryCreator] Cannot load adapterImpl class for auth-server-undertow
16:42:47,687 WARN  [org.keycloak.testsuite.arquillian.containers.RegistryCreator] Cannot load adapterImpl class for auth-server-remote
16:42:47,695 WARN  [org.keycloak.testsuite.arquillian.containers.RegistryCreator] Cannot load adapterImpl class for auth-server-jboss-migration
16:42:47,695 INFO  [org.keycloak.testsuite.arquillian.containers.RegistryCreator] Registering container: app-server-undertow
16:42:47,699 WARN  [org.keycloak.testsuite.arquillian.containers.RegistryCreator] Cannot load adapterImpl class for auth-server-undertow-backend1
16:42:47,700 WARN  [org.keycloak.testsuite.arquillian.containers.RegistryCreator] Cannot load adapterImpl class for auth-server-undertow-backend2
16:42:47,700 INFO  [org.keycloak.testsuite.arquillian.containers.RegistryCreator] Container is disabled: auth-server-undertow-backend1
16:42:47,701 INFO  [org.keycloak.testsuite.arquillian.containers.RegistryCreator] Container is disabled: auth-server-undertow-backend2
16:42:47,701 INFO  [org.keycloak.testsuite.arquillian.containers.RegistryCreator] Container is disabled: auth-server-balancer-undertow
16:42:47,703 WARN  [org.keycloak.testsuite.arquillian.containers.RegistryCreator] Cannot load adapterImpl class for cache-server-cross-dc-1
16:42:47,704 WARN  [org.keycloak.testsuite.arquillian.containers.RegistryCreator] Cannot load adapterImpl class for cache-server-cross-dc-2
16:42:47,705 INFO  [org.keycloak.testsuite.arquillian.containers.RegistryCreator] Container is disabled: auth-server-balancer-cross-dc
16:42:47,705 INFO  [org.keycloak.testsuite.arquillian.containers.RegistryCreator] Container is disabled: auth-server-undertow-cross-dc-0_1
16:42:47,705 INFO  [org.keycloak.testsuite.arquillian.containers.RegistryCreator] Container is disabled: auth-server-undertow-cross-dc-0_2-manual
16:42:47,706 INFO  [org.keycloak.testsuite.arquillian.containers.RegistryCreator] Container is disabled: auth-server-undertow-cross-dc-1_1
16:42:47,706 INFO  [org.keycloak.testsuite.arquillian.containers.RegistryCreator] Container is disabled: auth-server-undertow-cross-dc-1_2-manual
16:42:47,707 WARN  [org.keycloak.testsuite.arquillian.containers.RegistryCreator] Cannot load adapterImpl class for auth-server-jboss-cross-dc-0_1
16:42:47,709 WARN  [org.keycloak.testsuite.arquillian.containers.RegistryCreator] Cannot load adapterImpl class for auth-server-jboss-cross-dc-0_2-manual
16:42:47,711 WARN  [org.keycloak.testsuite.arquillian.containers.RegistryCreator] Cannot load adapterImpl class for auth-server-jboss-cross-dc-1_1
16:42:47,713 WARN  [org.keycloak.testsuite.arquillian.containers.RegistryCreator] Cannot load adapterImpl class for auth-server-jboss-cross-dc-1_2-manual
Configuration of Arquillian Reporting extension:
report                                   xml
rootDir                                  target
file                                     target/arquillian_report.xml
template                                 template.xsl
reportAfterEvery                         class
language                                 en
maxImageWidth                            500
imageWidth                               100%
imageHeight                              100%

Configuration of Arquillian Browser Screenshooter:
rootDir                                  target/screenshots
screenshotType                           PNG
takeBeforeTest                           false
takeAfterTest                            false
takeWhenTestFailed                       true
takeOnEveryAction                        false

16:42:47,719 INFO  [org.keycloak.testsuite.drone.HtmlUnitScreenshots] Screenshots disabled as htmlUnit is used
16:42:48,292 INFO  [org.keycloak.testsuite.arquillian.AuthServerTestEnricher] 

SUITE CONTEXT:
Auth server: auth-server-undertow

16:42:48,294 INFO  [org.keycloak.testsuite.arquillian.AuthServerTestEnricher] 

TEST PROCESS INFO: 
Available processors: 8
Total memory: 619 MB
Max memory (Xmx): 1820 MB
Metaspace Max: 256 MB
Sep 10, 2019 4:42:48 PM org.jboss.arquillian.drone.webdriver.factory.remote.reusable.ReusedSessionPermanentFileStorage readStore
INFO: Reused session store is not available at /home/jon/.drone-webdriver-session-store, a new one will be created.
Sep 10, 2019 4:42:48 PM org.jboss.arquillian.container.impl.MapObject populate
WARNING: Configuration contain properties not supported by the backing object org.keycloak.testsuite.arquillian.undertow.KeycloakOnUndertowConfiguration
Unused property entries: {adapterImplClass=org.keycloak.testsuite.arquillian.undertow.KeycloakOnUndertow, enabled=true && ! false}
Supported property names: [bindHttpPortOffset, bindHttpsPort, keycloakConfigPropertyOverrides, route, bindHttpPort, bindHttpsPortOffset, remoteMode, workerThreads, resourcesHome, bindAddress]
16:42:48,336 INFO  [org.keycloak.testsuite.arquillian.undertow.KeycloakOnUndertowConfiguration] KeycloakOnUndertow will listen for http on port: 8180 and for https on port: 8543
Sep 10, 2019 4:42:48 PM org.jboss.arquillian.container.impl.MapObject populate
WARNING: Configuration contain properties not supported by the backing object org.keycloak.testsuite.arquillian.undertow.UndertowAppServerConfiguration
Unused property entries: {adapterImplClass=org.keycloak.testsuite.arquillian.undertow.UndertowAppServer, enabled=true}
Supported property names: [bindHttpPortOffset, bindHttpPort, bindAddress]
16:42:48,339 INFO  [org.keycloak.testsuite.arquillian.undertow.UndertowAppServerConfiguration] App server undertow will listen on port: 8280
16:42:48,339 DEBUG [org.keycloak.testsuite.arquillian.AuthServerTestEnricher] Starting auth server before suite
16:42:48,340 INFO  [org.keycloak.testsuite.arquillian.AuthServerTestEnricher] ************************auth-server-undertow*****************************************************************************
16:42:48,342 INFO  [org.keycloak.testsuite.arquillian.undertow.KeycloakOnUndertow] Starting auth server on undertow.
16:42:48,355 INFO  [org.keycloak.testsuite.KeycloakServer] Using jboss.server.data.dir /home/jon/Projects/personal/keycloak/testsuite/integration-arquillian/tests/base/target/keycloak-server-24420772154985824/data
16:42:48,355 INFO  [org.keycloak.testsuite.arquillian.undertow.KeycloakOnUndertow] Starting auth server on embedded Undertow on: http://0.0.0.0:8180
16:42:50,614 INFO  [org.infinispan.factories.GlobalComponentRegistry] ISPN000128: Infinispan version: Infinispan 'Infinity Minus ONE +2' 9.4.14.Final
16:42:51,744 INFO  [org.keycloak.connections.infinispan.DefaultInfinispanConnectionProviderFactory] Node name: node_293628, Site name: null
16:42:52,861 DEBUG [org.keycloak.connections.jpa.DefaultJpaConnectionProviderFactory] Initializing JPA connections
16:42:52,862 DEBUG [org.keycloak.connections.jpa.DefaultJpaConnectionProviderFactory] Database info: {databaseUrl=jdbc:h2:mem:test, databaseUser=SA, databaseProduct=H2 1.4.193 (2016-10-31), databaseDriver=H2 JDBC Driver 1.4.193 (2016-10-31)}
16:42:55,247 INFO  [org.keycloak.connections.jpa.updater.liquibase.LiquibaseJpaUpdaterProvider] Initializing database schema. Using changelog META-INF/jpa-changelog-master.xml
16:42:56,421 INFO  [org.keycloak.connections.jpa.updater.liquibase.LiquibaseJpaUpdaterProvider] Initializing database schema. Using changelog META-INF/example-changelog.xml
16:43:01,455 INFO  [org.keycloak.testsuite.arquillian.undertow.KeycloakOnUndertow] Auth server started in 13100ms on http://0.0.0.0:8180/auth
16:43:01,500 INFO  [org.keycloak.testsuite.arquillian.AppServerTestEnricher] 

TEST CONTEXT: org.keycloak.testsuite.javascript.JavascriptAdapterTest

Sep 10, 2019 4:43:02 PM org.jboss.arquillian.drone.webdriver.factory.HtmlUnitDriverFactory setClientOptions
INFO: Setting HtmlDriver web client options: {htmlunitCssEnabled=false, htmlunitHistoryPageCacheLimit=1}
16:43:02,551 INFO  [org.keycloak.testsuite.drone.KeycloakDronePostSetup] Configuring driver settings. implicitWait=750, pageLoadTimeout=10000
Sep 10, 2019 4:43:03 PM org.jboss.arquillian.drone.webdriver.binary.BinaryFilesUtils extract
INFO: Extracting zip file: /home/jon/.arquillian/drone/phantomjs/2.1.1/phantomjs-2.1.1-linux-x86_64.tar.bz2 to /home/jon/Projects/personal/keycloak/testsuite/integration-arquillian/tests/base/target/drone/2a490bb643d5fc7365526daea8cec7c9
Sep 10, 2019 4:43:03 PM org.arquillian.spacelift.Spacelift$SpaceliftInstance <init>
INFO: Initialized Spacelift from defaults, workspace: /home/jon/Projects/personal/keycloak/testsuite/integration-arquillian/tests/base, cache: /home/jon/.spacelift/cache
16:43:08,110 ERROR [org.keycloak.testsuite.javascript.JavascriptAdapterTest] [JavascriptAdapterTest] null() FAILED
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 21.188 s <<< FAILURE! - in org.keycloak.testsuite.javascript.JavascriptAdapterTest
[ERROR] org.keycloak.testsuite.javascript.JavascriptAdapterTest  Time elapsed: 21.186 s  <<< ERROR!
java.lang.RuntimeException: Could not create statement
	at org.jboss.arquillian.junit.Arquillian.methodBlock(Arquillian.java:313)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.jboss.arquillian.junit.Arquillian$2.evaluate(Arquillian.java:166)
	at org.jboss.arquillian.junit.Arquillian.multiExecute(Arquillian.java:350)
	at org.jboss.arquillian.junit.Arquillian.access$200(Arquillian.java:54)
	at org.jboss.arquillian.junit.Arquillian$3.evaluate(Arquillian.java:177)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.jboss.arquillian.junit.Arquillian.run(Arquillian.java:115)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:383)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:344)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:417)
Caused by: java.lang.IllegalStateException: Something bad happened when Drone was trying to download and prepare a binary. For more information see the cause.
	at org.jboss.arquillian.drone.webdriver.binary.handler.AbstractBinaryHandler.checkAndSetBinary(AbstractBinaryHandler.java:63)
	at org.jboss.arquillian.drone.webdriver.factory.PhantomJSDriverFactory.getCapabilities(PhantomJSDriverFactory.java:99)
	at org.jboss.arquillian.drone.webdriver.factory.PhantomJSDriverFactory.createInstance(PhantomJSDriverFactory.java:72)
	at org.jboss.arquillian.drone.webdriver.factory.PhantomJSDriverFactory.createInstance(PhantomJSDriverFactory.java:37)
	at org.jboss.arquillian.drone.webdriver.factory.WebDriverFactory.createInstance(WebDriverFactory.java:127)
	at org.jboss.arquillian.drone.webdriver.factory.WebDriverFactory.createInstance(WebDriverFactory.java:38)
	at org.jboss.arquillian.drone.impl.DroneConfigurator$1.createInstance(DroneConfigurator.java:112)
	at org.jboss.arquillian.drone.impl.CachingCallableImpl.call(CachingCallableImpl.java:44)
	at org.jboss.arquillian.core.impl.threading.ThreadedExecutorService$ContextualCallable.call(ThreadedExecutorService.java:88)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.arquillian.spacelift.execution.ExecutionException: Execution of a task failed. Unable to execute task UntarTool
	at org.arquillian.spacelift.execution.ExecutionException.prependMessage(ExecutionException.java:63)
	at org.arquillian.spacelift.execution.impl.FutureBasedExecution.unwrapException(FutureBasedExecution.java:71)
	at org.arquillian.spacelift.execution.impl.FutureBasedExecution.await(FutureBasedExecution.java:130)
	at org.jboss.arquillian.drone.webdriver.binary.BinaryFilesUtils.extract(BinaryFilesUtils.java:58)
	at org.jboss.arquillian.drone.webdriver.binary.handler.PhantomJSDriverBinaryHandler.prepare(PhantomJSDriverBinaryHandler.java:43)
	at org.jboss.arquillian.drone.webdriver.binary.handler.AbstractBinaryHandler.downloadAndPrepare(AbstractBinaryHandler.java:227)
	at org.jboss.arquillian.drone.webdriver.binary.handler.AbstractBinaryHandler.downloadAndPrepare(AbstractBinaryHandler.java:209)
	at org.jboss.arquillian.drone.webdriver.binary.handler.AbstractBinaryHandler.downloadAndPrepare(AbstractBinaryHandler.java:176)
	at org.jboss.arquillian.drone.webdriver.binary.handler.AbstractBinaryHandler.checkAndSetBinary(AbstractBinaryHandler.java:61)
	... 12 more
Caused by: java.io.IOException: unexpected end of stream
	at org.apache.commons.compress.compressors.bzip2.BZip2CompressorInputStream.getAndMoveToFrontDecode(BZip2CompressorInputStream.java:726)
	at org.apache.commons.compress.compressors.bzip2.BZip2CompressorInputStream.initBlock(BZip2CompressorInputStream.java:312)
	at org.apache.commons.compress.compressors.bzip2.BZip2CompressorInputStream.setupNoRandPartA(BZip2CompressorInputStream.java:846)
	at org.apache.commons.compress.compressors.bzip2.BZip2CompressorInputStream.setupNoRandPartB(BZip2CompressorInputStream.java:893)
	at org.apache.commons.compress.compressors.bzip2.BZip2CompressorInputStream.read0(BZip2CompressorInputStream.java:218)
	at org.apache.commons.compress.compressors.bzip2.BZip2CompressorInputStream.read(BZip2CompressorInputStream.java:173)
	at org.apache.commons.compress.archivers.tar.TarArchiveInputStream.read(TarArchiveInputStream.java:656)
	at org.arquillian.spacelift.task.archive.UncompressTool.process(UncompressTool.java:120)
	at org.arquillian.spacelift.task.archive.UncompressTool.process(UncompressTool.java:36)
	at org.arquillian.spacelift.task.Task.run(Task.java:100)
	at org.arquillian.spacelift.task.Task$1.call(Task.java:71)
	... 4 more

16:43:08,115 INFO  [org.keycloak.testsuite.arquillian.undertow.KeycloakOnUndertow] Stopping auth server.
[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Errors: 
[ERROR]   JavascriptAdapterTest.org.keycloak.testsuite.javascript.JavascriptAdapterTest » Runtime
[INFO] 
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] Keycloak Arquillian Integration TestSuite 8.0.0-SNAPSHOT SUCCESS [  1.200 s]
[INFO] Test apps .......................................... SUCCESS [  0.171 s]
[INFO] Test apps distribution ............................. SUCCESS [  2.109 s]
[INFO] Keycloak Authz: PhotoZ  Test Parent ................ SUCCESS [  0.147 s]
[INFO] Keycloak Authz Test: Photoz RESTful API ............ SUCCESS [  1.689 s]
[INFO] Keycloak Authz Tests: Photoz HTML5 Client .......... SUCCESS [  0.678 s]
[INFO] Keycloak Authz Tests: Photoz Authz Rule-based Policy SUCCESS [  0.404 s]
[INFO] Keycloak Authz Tests: Hello World Example .......... SUCCESS [  0.420 s]
[INFO] Keycloak Authz: Servlet Authorization Test ......... SUCCESS [  0.493 s]
[INFO] Keycloak Authz: Simple Servlet App with Policy Enforcer SUCCESS [  0.400 s]
[INFO] integration-arquillian-test-apps-servlets .......... SUCCESS [  0.871 s]
[INFO] Keycloak Test App Profile JEE ...................... SUCCESS [  0.514 s]
[INFO] integration-arquillian-test-apps-cors-parent ....... SUCCESS [  0.146 s]
[INFO] Angular Product Portal JS .......................... SUCCESS [  0.933 s]
[INFO] JAX-RS Database Service Using OAuth Bearer Tokens .. SUCCESS [  0.528 s]
[INFO] Fuse Test Applications ............................. SUCCESS [  0.148 s]
[INFO] Customer Portal - Secured in Karaf/Fuse ............ SUCCESS [  1.628 s]
[INFO] CXF JAXRS Example - Secured in Karaf/Fuse .......... SUCCESS [  0.835 s]
[INFO] CXF JAXRS Example - Secured in Karaf/Fuse 7 on Undertow SUCCESS [  0.731 s]
[INFO] CXF JAXWS Example - Secured in Karaf/Fuse .......... SUCCESS [  0.704 s]
[INFO] CXF JAXWS Example - Secured in Karaf/Fuse 7 on Undertow SUCCESS [  0.706 s]
[INFO] Product Portal - Secured in Karaf/Fuse ............. SUCCESS [  0.740 s]
[INFO] Product Portal - Secured in Karaf/Fuse 7 on Undertow SUCCESS [  1.160 s]
[INFO] Camel endpoint example - Secured in Karaf/Fuse ..... SUCCESS [  0.723 s]
[INFO] Camel endpoint example - Secured in Karaf/Fuse 7.0 on Undertow SUCCESS [  0.636 s]
[INFO] Keycloak Fuse Example - Features ................... SUCCESS [  0.522 s]
[INFO] Keycloak Examples - External Config ................ SUCCESS [  0.640 s]
[INFO] spring-boot-adapter-app ............................ SUCCESS [  0.701 s]
[INFO] Servers ............................................ SUCCESS [  0.143 s]
[INFO] Auth Server ........................................ SUCCESS [  0.145 s]
[INFO] Auth Server Services ............................... SUCCESS [  0.150 s]
[INFO] Auth Server Services - Testsuite Providers ......... SUCCESS [  2.656 s]
[INFO] Auth Server - JBoss ................................ SUCCESS [  0.149 s]
[INFO] Test Util .......................................... SUCCESS [  1.013 s]
[INFO] Auth Server - Undertow ............................. SUCCESS [  1.908 s]
[INFO] App Server ......................................... SUCCESS [  0.145 s]
[INFO] App Server - SPI ................................... SUCCESS [  0.449 s]
[INFO] App Server - JBoss ................................. SUCCESS [  0.144 s]
[INFO] App Server - Karaf ................................. SUCCESS [  0.141 s]
[INFO] App Server - Tomcat ................................ SUCCESS [  0.145 s]
[INFO] App Server - Undertow .............................. SUCCESS [  0.755 s]
[INFO] App Server - Jetty Parent .......................... SUCCESS [  0.147 s]
[INFO] Cache Server ....................................... SUCCESS [  0.139 s]
[INFO] Cache Server - JBoss Family ........................ SUCCESS [  0.142 s]
[INFO] Tests .............................................. SUCCESS [  0.223 s]
[INFO] Base TestSuite ..................................... FAILURE [ 43.212 s]
[INFO] Other Tests Modules ................................ SKIPPED
[INFO] Adapter Tests ...................................... SKIPPED
[INFO] Adapter Tests - JBoss .............................. SKIPPED
[INFO] Adapter Tests - Karaf .............................. SKIPPED
[INFO] Adapter Tests - WAS ................................ SKIPPED
[INFO] Adapter Tests - WLS ................................ SKIPPED
[INFO] SSSD tests 8.0.0-SNAPSHOT .......................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:13 min
[INFO] Finished at: 2019-09-10T16:43:08+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.22.0:test (default-test) on project integration-arquillian-tests-base: There are test failures.
[ERROR] 
[ERROR] Please refer to /home/jon/Projects/personal/keycloak/testsuite/integration-arquillian/tests/base/target/surefire-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date]-jvmRun[N].dump, [date].dumpstream and [date]-jvmRun[N].dumpstream.
[ERROR] -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.22.0:test (default-test) on project integration-arquillian-tests-base: There are test failures.

Please refer to /home/jon/Projects/personal/keycloak/testsuite/integration-arquillian/tests/base/target/surefire-reports for the individual test results.
Please refer to dump files (if any exist) [date]-jvmRun[N].dump, [date].dumpstream and [date]-jvmRun[N].dumpstream.
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:213)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:154)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:146)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:954)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:288)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:192)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:289)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:229)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:415)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:356)
Caused by: org.apache.maven.plugin.MojoFailureException: There are test failures.

Please refer to /home/jon/Projects/personal/keycloak/testsuite/integration-arquillian/tests/base/target/surefire-reports for the individual test results.
Please refer to dump files (if any exist) [date]-jvmRun[N].dump, [date].dumpstream and [date]-jvmRun[N].dumpstream.
    at org.apache.maven.plugin.surefire.SurefireHelper.throwException (SurefireHelper.java:240)
    at org.apache.maven.plugin.surefire.SurefireHelper.reportExecution (SurefireHelper.java:112)
    at org.apache.maven.plugin.surefire.SurefirePlugin.handleSummary (SurefirePlugin.java:364)
    at org.apache.maven.plugin.surefire.AbstractSurefireMojo.executeAfterPreconditionsChecked (AbstractSurefireMojo.java:1052)
    at org.apache.maven.plugin.surefire.AbstractSurefireMojo.execute (AbstractSurefireMojo.java:868)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:208)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:154)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:146)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:954)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:288)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:192)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:289)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:229)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:415)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:356)
[ERROR] 
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :integration-arquillian-tests-base
```
</p>
</details>

Are you getting some other output?

--- a/adapters/oidc/js/src/main/resources/keycloak.d.ts
+++ b/adapters/oidc/js/src/main/resources/keycloak.d.ts
@@ -134,6 +134,12 @@ declare namespace Keycloak {
 		 * If not configured, PKCE will not be used.
 		 */
 		pkceMethod?: KeycloakPkceMethod;
+
+		/**
+		 * Disables any information from being logged to the console.
+		 * @default false
+		 */
+		disableLogging?: boolean
 	}
 
 	interface KeycloakLoginOptions {

--- a/adapters/oidc/js/src/main/resources/keycloak.js
+++ b/adapters/oidc/js/src/main/resources/keycloak.js
@@ -75,7 +75,9 @@
         }
 
         var useNonce = true;
-
+        var logInfo = createLogger(console.info);
+        var logWarn = createLogger(console.warn);
+        
         kc.init = function (initOptions) {
             kc.authenticated = false;
 
@@ -161,6 +163,9 @@
                     kc.pkceMethod = initOptions.pkceMethod;
                 }
 
+                if (typeof initOptions.disableLogging === 'boolean') {
+                    kc.disableLogging = initOptions.disableLogging;
+                }
             }
 
             if (!kc.responseMode) {
@@ -557,7 +562,7 @@
             }
 
             if (kc.timeSkew == null) {
-                console.info('[KEYCLOAK] Unable to determine if token is expired as timeskew is not set');
+                logInfo('[KEYCLOAK] Unable to determine if token is expired as timeskew is not set');
                 return true;
             }
 
@@ -585,10 +590,10 @@
                 var refreshToken = false;
                 if (minValidity == -1) {
                     refreshToken = true;
-                    console.info('[KEYCLOAK] Refreshing token: forced refresh');
+                    logInfo('[KEYCLOAK] Refreshing token: forced refresh');
                 } else if (!kc.tokenParsed || kc.isTokenExpired(minValidity)) {
                     refreshToken = true;
-                    console.info('[KEYCLOAK] Refreshing token: token expired');
+                    logInfo('[KEYCLOAK] Refreshing token: token expired');
                 }
 
                 if (!refreshToken) {
@@ -616,7 +621,7 @@
                         req.onreadystatechange = function () {
                             if (req.readyState == 4) {
                                 if (req.status == 200) {
-                                    console.info('[KEYCLOAK] Token refreshed');
+                                    logInfo('[KEYCLOAK] Token refreshed');
 
                                     timeLocal = (timeLocal + new Date().getTime()) / 2;
 
@@ -629,7 +634,7 @@
                                         p.setSuccess(true);
                                     }
                                 } else {
-                                    console.warn('[KEYCLOAK] Failed to refresh token');
+                                    logWarn('[KEYCLOAK] Failed to refresh token');
 
                                     if (req.status == 400) {
                                         kc.clearToken();
@@ -760,7 +765,7 @@
                     (kc.refreshTokenParsed && kc.refreshTokenParsed.nonce != oauth.storedNonce) ||
                     (kc.idTokenParsed && kc.idTokenParsed.nonce != oauth.storedNonce))) {
 
-                    console.info('[KEYCLOAK] Invalid nonce, clearing token');
+                    logInfo('[KEYCLOAK] Invalid nonce, clearing token');
                     kc.clearToken();
                     promise && promise.setError();
                 } else {
@@ -967,11 +972,11 @@
                 }
 
                 if (kc.timeSkew != null) {
-                    console.info('[KEYCLOAK] Estimated time difference between browser and server is ' + kc.timeSkew + ' seconds');
+                    logInfo('[KEYCLOAK] Estimated time difference between browser and server is ' + kc.timeSkew + ' seconds');
 
                     if (kc.onTokenExpired) {
                         var expiresIn = (kc.tokenParsed['exp'] - (new Date().getTime() / 1000) + kc.timeSkew) * 1000;
-                        console.info('[KEYCLOAK] Token expires in ' + Math.round(expiresIn / 1000) + ' s');
+                        logInfo('[KEYCLOAK] Token expires in ' + Math.round(expiresIn / 1000) + ' s');
                         if (expiresIn <= 0) {
                             kc.onTokenExpired();
                         } else {
@@ -1674,6 +1679,14 @@
             }
 
             return new CookieStorage();
+        }
+
+        function createLogger(fn) {
+            return function() {
+                if (!kc.disableLogging) {
+                    fn.apply(console, Array.prototype.slice.call(arguments));
+                }
+            };
         }
     }
 

